json-rpc/src/views.rs

```diff
@@ -221,7 +221,7 @@ impl From<Transaction> for TransactionDataView {
                     timestamp_usecs: x.1,
                 })
             }
-            Transaction::WriteSet(_) => Ok(TransactionDataView::WriteSet {}),
+            Transaction::AuthenticatedWriteSet(_) => Ok(TransactionDataView::WriteSet {}),
```

Comment:
@phoenix-antigravity @thefallentree should we rename `TransactionDataView::WriteSet` to `TransactionDataView::AuthenticatedWriteSet` as well?